### ISSUE TITLE
@messageformat/cli v3.0.0 broken argument parsing

### ISSUE THREAD

**Issue by aeirola** [messageformat/messageformat#323]
It seems as if the argument parsing is somewhat broken in the cli. Running the command

```sh
npx --package @messageformat/cli@3.0.0 -- messageformat file.json
```

gives the error

```
Cannot read property 'reduce' of undefined
```

On the other hand, if I pass `locale` and `options` parameters, even if they are empty, it does seem to work.

```sh
npx --package @messageformat/cli@3.0.0 -- messageformat file.json --locale --options
```

In the last beta version, 3.0.0-beta.2, everything still worked normally. Maybe this has something to do with [updating yargs](https://github.com/messageformat/messageformat/commit/142d66f65636a5d99e7a6fb69d1c1bd27c71dd02) before the 3.0.0 release?

**Comment by eemeli** [comment]
Sorry for not getting to this earlier; I wasn't able to replicate this initially, so put it on a back-burner. Turns out that my local `node_modules` folder was a bit corrupted, and I was using `yargs@16` even though my `package.json` and lockfile claimed I had `yargs@17`.

In any case, there was indeed a breaking change in the yargs update that had not been accounted for.

**Comment by eemeli** [comment]
Fix now available in @messageformat/cli 3.0.1.
